backend/docker-compose.yml

```
@@ -0,0 +1,52 @@
+      - postgres_data:/var/lib/postgresql/data
+    networks:
+      - app_network
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U ${POSTGRES_USER} -d ${POSTGRES_DB}"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+  # Service backend Spring Boot
+  backend:
+    build: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)